examples/simple/lint/L-EXC3-bad.c

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+
+#include <stdlib.h>
+#include <stdio.h>
+
+void * func()
+{
+    int * retval = malloc(sizeof(int));
+    *retval = 1;
+    return retval; 
```

Comment:
Malloc has been replaced by a statically allocated object in both -good and -bad versions.